packages/bot-web-ui/src/components/save-modal.jsx

```diff
@@ -0,0 +1,161 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Checkbox, Icon, Modal, RadioGroup } from '@deriv/components';
+import { Formik, Form, Field } from 'formik';
+import { localize } from '@deriv/translations';
+import { connect } from '../stores/connect';
+import '../assets/sass/google-drive.scss';
+import '../assets/sass/save-modal.scss';
+
+const initial_option = { is_local: true, save_as_collection: false };
+
+const SaveModal = ({
+    button_status,
+    is_authorised,
+    is_save_modal_open,
+    onConfirmSave,
+    onDriveConnect,
+    toggleSaveModal,
+}) => (
+    <Modal
+        title={'Save Strategy'}
+        className='modal--save'
+        width='384px'
+        is_open={is_save_modal_open}
+        toggleModal={toggleSaveModal}
+    >
+        <Formik initialValues={initial_option} onSubmit={onConfirmSave}>
+            {({ values: { is_local, save_as_collection }, setFieldValue }) => (
+                <Form>
+                    <div className='modal__content'>
+                        <div className='modal__content-row'>
+                            <RadioGroup
+                                className='radio-group__save-type'
+                                name='is_local'
+                                items={[
+                                    {
+                                        id: 'local',
+                                        label: (
+                                            <IconRadio
+                                                text={localize('Local')}
+                                                icon={<Icon icon='IcDesktop' size={48} custom_color='#2A3052' />}
```

Comment:
Could we add this custom colour to shared styles, e.g. `constants.scss`?

Reply:
seems like the props `custom_color` used as an inline style which doesn't accept variable css. You can refer to `components/icon.jsx`